web/src/features/weather-layers/wind-layer/windy.ts

```diff
@@ -532,8 +532,8 @@ var Windy = function (params: any) {
     started: any;
     field?: any;
     params?: any;
-    start?: (bounds: any, width: any, height: any, extent: any) => void;
-    stop?: () => void;
+    start: (bounds: any, width: any, height: any, extent: any) => void;
```

Comment:
I really hope width and height are numbers 😅

Reply:
Haha yes, but I don't think it's worth typing this file up without refactoring it completely - and I don't think that's the best use of our time for now :D